src/services/data_model_engine.js

```diff
@@ -281,4 +281,12 @@ export default class DataModelEngine {
   async getWorkerLogs(logsCount = 10) {
     return await this.workerLogRepository.getLogs(logsCount);
   }
+
+  async areLogsRecent(maximalLogAgeInSeconds) {
+    const logs = await this.workerLogRepository.getLogs(1);
+    if (logs.length === 0) {
+      return false;
+    }
+    return Date.now() - logs[0].timestamp <= maximalLogAgeInSeconds * 1000;
+  }
```

Comment:
I'd go for 
```
    if (logs.length > 0) {
       return Date.now() - logs[0].timestamp <= maximalLogAgeInSeconds * 1000;
    }
   return false;
```
Then . works correctly not only for `0`, but also `null` or `undefined` (just in case)